knossos_utils/knossosdataset.py

```diff
@@ -797,6 +797,8 @@ def initialize_without_conf(self, path, boundary, scale, experiment_name,
                                         all_mag_folders[0][:-1])[-1] + str(mag))
                         else:
                             os.makedirs(path+"/mag"+str(mag))
+        else:
```

Comment:
In case of `mags is not None and not make_mag_folders`, the assert would not ensure that mags exist.
Can you either remove the `else:` or combine the `make_mag_folders` check with the `mags is None` check further above? To ensure that your assert always happens.

Reply:
I implemented the last suggestion: If default value for `mags` (`mags is None`)  is used, and the creation of magnification folders is set to `False`, a magnification as to exist already.

Reply:
Thx